numba/cuda/compiler.py

```diff
@@ -317,6 +322,9 @@ def bind(self):
 
     @property
     def ptx(self):
+        '''
+        Returns the PTX code for this kernel encoded in UTF-8.
```

Comment:
As it's a property rather than a function I decided to drop the word "return" entirely, and removed the statement that it's encoded in UTF-8.